sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -573,20 +527,6 @@ private static async Task DownloadAndAssertAsync(Stream stream, BlobClient blob)
             // Use a 1KB threshold so we get a lot of individual blocks
             await UploadStreamAndVerify(size, Constants.KB, new StorageTransferOptions { MaximumTransferLength = Constants.KB });
 
-        [Test]
```

Comment:
Why are you removing all these tests?   We should probably revert all the changes in this file.

Reply:
I identified an unused internal method in the SDK, and these tests are for that method. I discussed removing that method with you and you were onboard.